### PR TITLE
Regenerate backend test cert to include additional SANs

### PR DESCRIPTION
**Pull request by treydock**
# Pull Request Checklist

## Description

Added `127.0.0.1` and `localhost` to subject alt names for backend certs.

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Could you please document how the SSL certs are updated

**Comment by treydock** [comment]
Added `README.md` to `tests` directory that documents how certs were generated

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+```
+puppet cert generate sensu_backend --dns_alt_names=localhost,127.0.0.1,sensu_backend,sensu_backend1,sensu_backend2,sensu_backend3,sensu-backend.example.com,sensu-backend-peer1.example.com,sensu-backend-peer2.example.com
+puppet cert generate sensu_agent
+```
+
+Copy certs from vagrant instance to this repo
+
+```
+\cp -r /etc/puppetlabs/puppet/ssl/* /vagrant/tests/ssl/
```
Comment (ghoneycutt) [inline tests/README.md]: `\` before `cp` looks like a typo

Comment (treydock) [reply tests/README.md]: that's actually deliberate, it avoids the default alias of `cp -i` which is annoying when re-generating certificates and trying to overwrite old ones.  Using `cp -rf` doesn't help so have to use `\` to bypass the alias.

**Pull request merged**